src/styles/_card.scss

```
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  height: 408px;
+  background-color: white;
+  &__photo {
+    width: 160px;
+    height: 134px;
+    padding: 33px 20px 0;
```

to make outer indent between elements you need to use margin instead of padding
```suggestion
    margin: 0 auto 40px;
```